schemas/upgrade.json

```diff
@@ -22,6 +22,14 @@
             "sectionBegin": "Upgrade",
             "propertyOrder": 10
         },
+        "vsd_pre_upgrade_check_script": {
+            "type": "string",
+            "title": "VSD Pre upgrade check script File",
+            "default": "",
+            "description": "Path to the VSD Pre upgrade check script file for the VSD",
```

Comment:
This is outside any section begin and section end. Needs to be Inside the section begin and section end so that the excel file sections it correctly.

Reply:
corrected